server/odc-server/src/main/java/com/oceanbase/odc/server/web/controller/v2/ScheduleController.java

```diff
@@ -121,6 +126,24 @@ public SuccessResponse<String> getTaskLog(@PathVariable Long scheduleId, @PathVa
         return Responses.success(scheduleService.getLog(scheduleId, taskId, logType));
     }
 
+    @ApiOperation(value = "GetFullLogDownloadUrl", notes = "获取下载全量日志的URL")
```

Comment:
avoid non-english, even in swagger annotation

Reply:
ok